### PR TITLE
upgrade go to 1.19.x

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- upgrade go to 1.19.x

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,7 +61,7 @@ GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint
 golangci-lint:
 	rm -f $(GOLANGCI_LINT) || :
 	set -e ;\
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell dirname $(GOLANGCI_LINT)) v1.47.3 ;\
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell dirname $(GOLANGCI_LINT)) v1.50.0 ;\
```
Comment (sabre1041) [inline Makefile:64]: Can you also update the version in the `Makefile`?

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Quick request on aligning golang lint versions

File: Makefile (hunk 1 above)
Comment (cpanato) [reply Makefile:64]: i did, am i missing something else?

Comment (sabre1041) [reply Makefile:64]: Apologies. Long morning

**Review by sabre1041 — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (cpanato) [reply Makefile:64]: all good :)

**Pull request merged**